rootfs/api/models/app.py

```diff
@@ -657,7 +681,7 @@ def verify_application_health(self, **kwargs):
             time.sleep(1)
 
         # Endpoint did not report healthy in time
-        if ('response' in locals() and response.status_code == 404) or failed:
+        if (('response' in locals() and response.status_code == 404) or failed) and not is_experimental_native_ingress():
```

Comment:
Ha. This is funny. This would wait for the app to become available, right? But if experimental ingress is enabled, the possibility exists that no ingress controller is installed...

@mboersma should make the call on this, but I think it might be better to leave this the way it was and instead make the documentation crystal clear that if you want to BYO ingress controller, you need to have that installed _before_ you can rightfully expect Workflow to start working.

I'm just worried about what happens if we totally give up this idea of waiting for the app to be up and responding.